bot/modules/bartek/text_handler.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import os
+
+
+class TextHandler:
+    def __init__(self, log: logging.Logger, path: str) -> None:
+        self.log: logging.Logger = log
+
+        self.blacklist: set[str] = set()
+
+        for file_name in os.listdir(path):
+            with open(path + file_name, "r") as file:
+                for line in file.readlines():
+                    file_content = line.split(",")
+                    file_content = filter(
+                        lambda x: x != None and x != "\n",
```

Comment:
```py
lambda x: x and x != "\n"
```
would also filter out empty elements, which can appear if there is a double comma (`,,`) somewhere by mistake